dags/general/monitoring.py

```diff
@@ -57,6 +51,17 @@ def cleanup_xcom(**context):
     resolve_stitch_status = PythonOperator(
         task_id='resolve_stitch_status', provide_context=True, python_callable=resolve_stitch
     )
+    hightouch_check_syncs = SimpleHttpOperator(
+        task_id="check_hightouch_syncs",
+        http_conn_id="hightouch",
+        method="GET",
+        endpoint=Variable.get('hightouch_syncs_endpoint'),
+        headers={'Content-Type': 'application/json', 'Authorization': Variable.get('hightouch_secret')},
+        xcom_push=True,
+    )
+    resolve_hightouch_status = PythonOperator(
+        task_id='resolve_hightouch_status', provide_context=True, python_callable=resolve_hightouch
+    )
```

Comment:
The two operators for hightouch are not wired together.

Reply:
setting downstream